src/main/resources/assets/presencefootsteps/lang/de_de.json

```diff
@@ -0,0 +1,67 @@
+	"menu.pf.global.players_and_hostiles": "Entitäten: Spieler + Feindliches",
+	"menu.pf.global.players_only": "Entitäten: Nur Spieler",
+	"menu.pf.volume.min": "Gesamtlautstärke: AUS",
+	"menu.pf.volume": "Gesamtlautstärke: %d%%",
+	"menu.pf.volume.tooltip": "Die Gesamtlautstärke aller Presence Footsteps Geräusche. Setze auf 0 wenn du die Mod deaktivieren möchtest.",
+	"menu.pf.volume.hostile_entities": "Feindliche Entitäten: %d%%",
+	"menu.pf.volume.hostile_entities.tooltip": "Ändert die Lautstärle von feindlichen Laufgeräuschen.",
+	"menu.pf.volume.passive_entities": "Neutrale Entitäten: %d%%",
+	"menu.pf.volume.passive_entities.tooltip": "Ändert die Lautstärke von neutralen Laufgeräuschen.",
+	"menu.pf.volume.player": "Eigener Spieler: %d%%",
+	"menu.pf.volume.player.tooltip": "Ändert die Lautstärke des eigenen Spielers.",
+	"menu.pf.volume.other_players": "Andere Spieler: %d%%",
+	"menu.pf.volume.other_players.tooltip": "Ändert die Lautstärke der Laufgeräuschen anderer Spieler im Multiplayer.",
+	"menu.pf.volume.running": "Geschwindigkeitsanpassung: %d%%",
+	"menu.pf.volume.running.tooltip": "Wie stark die Lautstärke der Laufgeräusche beim Rennen variieren kann.\n\nDieser Wert wird zur Gesamtlautstärke HINZUGEFÜGT und skaliert mit Ihrer Geschwindigkeit.\n\nEs empfiehlt sich bei kleinen Werten zu starten und langsam anzupassen. Größere Werte verstärken den Effekt.",
+	"menu.pf.volume.wet": "Nasse Oberflächen: %s",
+	"menu.pf.volume.wet.tooltip": "Ändert die Lautstärke der Geräusche, die beim Gehen auf nassen Blöcken entstehen.",
+	"menu.pf.stance": "Gangart: %s",
+	"menu.pf.stance.auto": "Automatisch",
+	"menu.pf.stance.none": "Keine",
+	"menu.pf.stance.none.tooltip": "Auto wählt basierend auf 'Mein kleines Pony' eine geeignete Haltung. Wenn dieser Mod nicht installiert ist, verhält er sich genauso wie Biped",
+	"menu.pf.stance.quadruped": "Vierbeiner",
+	"menu.pf.stance.quadruped.tooltip": "Vierbeiner emuliert eine vierbeinige Gangart",
+	"menu.pf.stance.biped": "Zweibeiner",
+	"menu.pf.stance.biped.tooltip": "Zweibeiner emuliert eine zweibeinige (menschliche) Gangart",
+	"menu.pf.stance.flying": "Pegasus",
+	"menu.pf.stance.flying.tooltip": "Pegasus emuliert einen vierbeinigen Gehmodus mit Flügeln",
+	"menu.pf.stance.flying_biped": "Vogel",
+	"menu.pf.stance.flying_biped.tooltip": "Vogel emuliert einen zweibeinigen Gehmodus mit Flügeln",
+	"menu.pf.report.full": "Kompletter Bericht",
+	"menu.pf.report.concise": "Kurzer Bericht",
+	"pf.report.save": "Datei gespeichert unter: %s",
+	"pf.report.error": "Bericht konnte nicht erstellt werden: %s",
+	"pf.update.title": "Ein Update ist verfügbar",
+	"pf.update.text": "Presence Footsteps %s",
+	"pf.update.checking": "Suche nach Updates...",
+	"pf.update.updates_available": "Ein Update ist verfügbar!\n Presence Footsteps %s für MC%s",
+	"pf.update.up_to_date": "Auf dem neusten Stand",
+	"subtitles.pf.footsteps": "Schritte",
+	"subtitles.pf.wood_squeak": "Holz quietschen",
```

Comment:
Holz quietscht
all of the following entries incorrectly use the infinitive of the verb instead of the third person singular
or a noun describing the action instead of the verb
or a not fitting verb

Reply:
Your right, this format will make it more describing thanks!